README.md

```diff
@@ -1 +1,58 @@
-# StockHook
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Stock Hook
+
+> Stock Hook app is all about watching the changes happens in stock market.
+
+## Built With
+
+- React JS
+- Redux
+- JavaScript (ES6)
+- [financialmodelingprep API](https://financialmodelingprep.com/)
+- Linters (Stylelint, Eslint)
+- Gitflow
+- Jest
+
+## Live demo
+
+You can view a live demo of the project using this [link]().
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- A web browser 
+- A code editor
+
+### Setup
+
+Assuming you have git and node.js installed on your computer, please follow the steps below: 
+
+- Clone the repository using the following command: `git clone https://github.com/KareemWilson/StockHook.git`
+- Navigate to that folder in the command line using the `cd` command.
+- Run the following command to install the necessary packages: `npm install`. 
+- Run the following command to view the project in the browser: `npm run start`. 
+
+## Authors
+
+👤 **Kareem Magdy Wilson**
+
+- GitHub: [@KareemWilson](https://github.com/KareemWilson)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/kareem-wilsons)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
```

Comment:
- [ ] It would be better if our README file has the acknowledgement section where we give appropriate credit to `Nelson Sakwa`, the author of the original design as required by `Creative Commons License`